watcher/lib/influxdb/influxdb.go

```diff
@@ -57,8 +61,18 @@ func (c *Client) Setup() error {
 	queries := []string{
 		fmt.Sprintf(createAdminQuery, constants.InfluxDBAdminUser, constants.InfluxDBAdminPassword),
 		fmt.Sprintf(createUserQuery, constants.InfluxDBGrafanaUser, constants.InfluxDBGrafanaPassword),
-		fmt.Sprintf(createDatabaseQuery, constants.InfluxDBDatabase, constants.DurationDefault),
+		fmt.Sprintf(createDatabaseQuery, constants.InfluxDBDatabase),
 		fmt.Sprintf(grantReadQuery, constants.InfluxDBDatabase, constants.InfluxDBGrafanaUser),
+	}
+	for _, query := range queries {
+		log.Infof("%v", query)
```

Comment:
nit: I'd be confused if I saw a log message without context, does not look too helpful.
I would also rephrase the doc-comment on Setup - it should be obvious what it does. Is it supposed to populate an initially empty database? What happens if it encounters existing data?